tests/apparmor-profiles/task.yaml

```diff
@@ -0,0 +1,30 @@
+summary: Check apparmor profile generation for apps and hooks.
+prepare: |
+  for snap in basic-binaries basic-hooks
+  do
+    snapbuild ./../fixtures/snaps/$snap .
+  done
+restore: |
+  for snap in basic-binaries basic-hooks
+  do
+    sudo snap remove $snap
+    rm ./${snap}_1.0_all.snap
```

Comment:
Removing the built file makes sense, but no need to remove the snap. The suite `restore-each` script is resetting the whole snapd state for every test.